src/_component/config/common.scss

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap');
```

Comment:
replace CSS import with SCSS import, ask on #general if you don't know the difference.
The current one will create an extra request to the server